lib/service/connect.go

```diff
@@ -552,6 +552,7 @@ func (process *TeleportProcess) firstTimeConnect(role types.SystemRole) (*Connec
 		Spec: state.StateSpecV2{
 			Rotation:            ca.GetRotation(),
 			InitialLocalVersion: teleport.Version,
+			LocalVersion:        teleport.Version,
```

Comment:
Hmm I'm not sure this is the correct place to ensure auth server has this version set in its local state upon first start. I think this code runs on agents. You might wanna look at Init in https://github.com/gravitational/teleport/blob/master/lib/auth/init.go.